src/SimpleThings/EntityAudit/EventListener/LogRevisionsListener.php

```diff
@@ -309,6 +309,16 @@ private function getRevisionId()
     }
 
     /**
+     * @return string|null
+     */
+    private function getUsername()
+    {
+        $callable = $this->config->getUsernameCallable();
+
+        return $callable ? $callable() : null;
```

Comment:
What if the "old style" username is required (where username was just assigned?) 

Suggestion: make the `$callable` return the configured username by default.